cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1076,13 +1208,37 @@ private boolean tryPullTimeSeriesSchemas(Node node, PullSchemaRequest request) {
             allTimeseriesName);
         result.addAll(allTimeseriesName);
       } else {
+
+        if (withAlias && clusterConfig.isEnableQueryPathsCache()) {
```

Comment:
if the `withAlias==false`? we will not use the cache?

Reply:
yes, The query block is calling the withAlias process, so we can only cache data for that process.